opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -503,9 +500,107 @@ message DoubleHistogramDataPoint {
   // a boolean value which decides what type of intervals to use.
   repeated double explicit_bounds = 7;
 
+  // When bounds have a pattern, a more efficient encoding method may be used.
```

Comment:
There may be a technicality that we need to add above. I'm interested in the zero "bucket" of an exponential bounds. If there are no negative values in the distribution, but there are zeros, how will it be represented? Do we need to insert an unused 0 boundary into the sequence so that the corresponding explicit bounds are `(-Inf, 0), [0, x)` and the 0th bucket goes unused?

Reply:
yes, you need set the has_counter_for_zero flag in exponential format so that the bound sequence starts from 0.